package.json

```diff
@@ -396,7 +396,7 @@
   },
   "packageManager": "pnpm@8.10.0",
   "volta": {
-    "node": "16.20.0",
+    "node": "18.20.6",
```

Comment:
Node 16 support was already dropped (https://github.com/emberjs/ember.js/pull/20716) so this simply updates the volta config to match that. 

Without this the content-tag package doesn't work correctly (Webassembly related errors).